docs/run-node/run-node.md

```diff
@@ -0,0 +1,90 @@
+<!--
+order: 2
+-->
+
+# Running a Node
+
+Now that the application is ready and the keyring populated, it's time to see how to run the blockchain node. In this section, the application we are running is called [`simapp`](https://github.com/cosmos/cosmos-sdk/tree/v0.40.0-rc2/simapp), and its corresponding CLI binary `simd`. {synopsis}
+
+## Pre-requisite Readings
+
+- [Anatomy of an SDK Application](../basics/app-anatomy.md) {prereq}
+- [Setting up the keyring](./keyring.md) {prereq}
+
+## Initialize the Chain
+
+::: warning
+Make sure you can build your own binary, and replace `simd` with the name of your binary in the snippets.
+:::
+
+Before actually running the node, we need to initialize the chain, and most importantly its genesis file. This is done with the `init` subcommand:
+
+```bash
+# The argument <moniker> is the custom username of your node, it should be human-readable.
+simd init <moniker> --chain-id my-test-chain
+```
+
+The command above creates all the configuration files needed for your node to run, as well as a default genesis file, which defines the initial state of the network. All these configuration files are in `~/.simapp` by default, but you can overwrite the location of this folder by passing the `--home` flag.
+
+The `~/.simapp` folder has the following structure:
+
+```bash
+.                                   # ~/.simapp
+  |- data                           # Contains the databases used by the node.
+  |- config/
+      |- app.toml                   # Application-related configuration file.
+      |- config.toml                # Tendermint-related configuration file.
+      |- genesis.json               # The genesis file.
+      |- node_key.json              # Tendermint node pubkey that will be used for signing blocks.
+      |- priv_validator_key.json    # TODO
```

Comment:
Does someone have a link to what this file does? In particular, I don't know the difference between this file and the `node_key.json` one.

Reply:
```
# Path to the JSON file containing the private key to use as a validator in the consensus protocol
priv_validator_key_file = "config/priv_validator_key.json"

# Path to the JSON file containing the private key to use for node authentication in the p2p protocol
node_key_file = "config/node_key.json"
```